cocos/gfx/base/device.ts

```diff
@@ -77,14 +77,6 @@ export abstract class Device {
         return this._cmdBuff as CommandBuffer;
     }
 
-    /**
-     * @en The current format of the swapchain being used.
-     * @zh 当前使用的swapchain的格式。
-     */
-    get swapchainFormat (): Format {
-        return this._swapchainFormat;
```

Comment:
gfx interfaces are public.
Will it  cause compatiblity issue if removing the getter?